### ISSUE TITLE
Server not added to dashboard every time

### ISSUE THREAD

**Issue by Omvk97** [jakobhviid/Dashboard-Interface-Docker#23]

**Comment by Omvk97** [comment]
If the interface does not find anything from kafka while running, it should try to go back in the kafka consumer and fetch the latest message instead

**Comment by dstoft** [comment]
I have fixed this in branch: https://github.com/jakobhviid/Dashboard-Interface-Docker/tree/feature/issue-23-overview-bug
However, I believe that some sort of UI change should happen, to specify if the Dashboard server is actually online, or if this is just the last message received from a Dashboard server that is now offline.
Maybe just show a timestamp of when the last update was received?

**Comment by dstoft** [comment]
Requires the implementation from https://github.com/jakobhviid/Dashboard-Server-Docker/issues/13 to not have an "undefined" timestamp.

I've added a small timestamp to the table container, when the table is in "server-mode".
![image](https://user-images.githubusercontent.com/22538335/102716973-97643880-42df-11eb-8b2e-9fe848af518e.png)

There is no way to tell right now if a server is online or not, so "only" this timestamp has been implemented.
